internal/vrf/create.go

```diff
@@ -0,0 +1,74 @@
+package vrf
+
+import (
+	"fmt"
+	"strconv"
+	"github.com/packethost/packngo"
+	"github.com/spf13/cobra"
+)
+
+
+func (c *Client) Create() *cobra.Command {
+	var metro, name, IPranges string, LocalASN int, description string
+
+	// createVRFCmd represents the creatVRF command
+	createVRFCmd := &cobra.Command{
+		Use:   `create vrf <my_vrf> [-m <metro_code>] [-AS int] [-I str] [-d <description>]`,
+		Short: "Creates a virtual network.",
+		Long:  "Creates a VRF",
+		Example: `# Creates a VRF, metal vrf create `,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			req := &packngo.VRFCreateRequest{
+				Metro:     metro,
+				name:	   name,
```

Comment:
In the `VRFCreateRequest` type, any fields that can be defined here must have been exported (made public). The `Name` field would have been one of those fields. The field name would be capitalized:

```suggestion
				Name:	   name,
```